app/src/main/java/org/mozilla/fenix/components/metrics/Metrics.kt

```diff
@@ -352,7 +352,7 @@ sealed class Event {
         enum class Item {
             SETTINGS, LIBRARY, HELP, DESKTOP_VIEW_ON, DESKTOP_VIEW_OFF, FIND_IN_PAGE, NEW_TAB,
             NEW_PRIVATE_TAB, SHARE, REPORT_SITE_ISSUE, BACK, FORWARD, RELOAD, STOP, OPEN_IN_FENIX,
-            SAVE_TO_COLLECTION, ADD_TO_FIREFOX_HOME, ADD_TO_HOMESCREEN, QUIT, READER_MODE_ON,
```

Comment:
Should we update this as well? https://github.com/mozilla-mobile/fenix/blob/master/app/metrics.yaml#L81

Reply:
done